kinesisfirehose-logs-extension-demo/agent/http.go

```diff
@@ -0,0 +1,168 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+package agent
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+
+	"aws-lambda-extensions/kinesisfirehose-logs-extension-demo/logsapi"
+
+	"github.com/golang-collections/go-datastructures/queue"
+)
+
+// DefaultHttpListenerPort is used to set the URL where the logs will be sent by Logs API
+const DefaultHttpListenerPort = "1234"
```

Comment:
Is it worth having this as a default which can be overwritten by an ENVAR?

Reply:
Added new environment variable `HTTP_LOGS_LISTENER_PORT` which allows customization of the default http ports for receiving log events